charts/merly-mentor/values.yaml

```diff
@@ -12,7 +13,7 @@ bridge:
   # nameOverride: ""  # optional - default "merly-bridge"
   image:
     repository: merlyai/merly-mentor-bridge
-    tag: v0.5.0
+    tag: Test_0.6.3
```

Comment:
@Saif-Uz-Zaman - can you use the latest `0.6.x` release in this chart? Since `Test` is listed here, I'm concerned it's not using our release channel.